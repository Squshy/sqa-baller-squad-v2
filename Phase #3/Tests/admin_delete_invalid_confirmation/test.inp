LOGIN
ADMINBOY
AdPass123
delete
test123
Kai
logout
exit
