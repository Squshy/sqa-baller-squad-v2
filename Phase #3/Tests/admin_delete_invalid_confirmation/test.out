=========================================================
                           DELETE
=========================================================

Enter username: Test123

Are you sure you want to delete user Test123? 
(Yes/No): KAI

---------------------------------------------------------
                Invalid Confirmation.
=========================================================